theano/sandbox/cuda/corr_gemm.cu

```diff
@@ -54,6 +54,7 @@ inline int GET_BLOCKS(const int N) {
 // Kernels for fast unfold + copy
 __global__ void im2col_kernel(const int n, const float* data_im,
     const int height, const int width, const int kernel_h, const int kernel_w,
+    const int dilation_h, const int dilation_w,
```

Comment:
Still: What about re-copying the code from Caffe instead of making our own modifications? Might make it easier to maintain. (https://github.com/BVLC/caffe/blob/master/src/caffe/util/im2col.cu)